core/entities/objective.go

```diff
@@ -97,3 +98,16 @@ func NewObjective() *Objective {
 		},
 	}
 }
+
+// turn indicator labels into a base64 encoded selector
+func (o *Objective) GetIndicatorsLabelsAsSelector() string {
+	var labels Selector = o.Spec.IndicatorSelector
+
+	var str string
+	for k, v := range labels {
+		str += fmt.Sprintf("%s=%s", k, v)
+	}
+
+	data := []byte(str)
+	return base64.StdEncoding.EncodeToString(data)
```

Comment:
```suggestion
	return base64.URLEncoding.EncodeToString(data)
```

The API expects Base64 URL encoding, in most cases there's no difference but for some special characters, it could cause problems.